web/src/routes/OrganizationsList/Organization/Tabs/DefaultPermissions/DefaultPermissionsList.tsx

```diff
@@ -57,14 +68,91 @@ export default function DefaultPermissionsList(
   ) => {
     setSelectedPermissions((prevSelected) => {
       const otherSelectedPermissions = prevSelected.filter(
-        (p) => p.createdBy !== permission.createdBy,
+        (p) => p.id !== permission.id,
       );
       return isSelecting
         ? [...otherSelectedPermissions, permission]
         : otherSelectedPermissions;
     });
   };
 
+  const mapOfColNamesToTableData = {
+    'Repository Created By': {
+      label: 'creator',
+      transformFunc: (perm: IDefaultPermission) => {
+        return `${perm.createdBy}`;
+      },
+    },
+    'Permission Applied To': {
+      label: 'members',
+      transformFunc: (perm: IDefaultPermission) => perm.appliedTo,
+    },
+    Permission: {
+      label: 'team role',
+      transformFunc: (perm: IDefaultPermission) => (
+        <Dropdown
+          toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+            <MenuToggle ref={toggleRef} id="toggle-disabled" isDisabled>
+              {perm.permission}
+            </MenuToggle>
+          )}
+          isOpen={false}
+        >
+          <DropdownList>
+            <DropdownItem>{perm.permission}</DropdownItem>
+          </DropdownList>
+        </Dropdown>
+      ),
+    },
+  };
+
+  const {bulkDeleteDefaultPermissions} = useBulkDeleteDefaultPermissions(
+    props.orgName,
+    {
+      onSuccess: () => {
+        setSelectedPermissions([]);
+        setBulkDeleteModalIsOpen(!bulkDeleteModalIsOpen);
+      },
+      onError: (err) => {
+        if (err instanceof BulkOperationError) {
+          const errMessages = [];
+          err.getErrors().forEach((error, perm) => {
+            errMessages.push(
+              addDisplayError(
+                `Failed to delete default permission created by: ${perm}`,
+                error.error,
+              ),
+            );
+          });
+          setIsError(errMessages);
+        } else {
+          setIsError([
+            addDisplayError('Failed to delete default permission', err),
+          ]);
+        }
+        setSelectedPermissions([]);
+        setBulkDeleteModalIsOpen(!bulkDeleteModalIsOpen);
+      },
+    },
+  );
+
+  const handleBulkDeleteModalToggle = () => {
+    setBulkDeleteModalIsOpen(!bulkDeleteModalIsOpen);
+  };
+
+  const deleteModal = (
```

Comment:
nit: Can this be added directly into the return statement? Having markup intertwined with JS makes it difficult to read IMO